webpack/JobWizard/steps/AdvancedFields/__tests__/AdvancedFields.test.js

```diff
@@ -1,26 +1,141 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import configureMockStore from 'redux-mock-store';
-import * as patternfly from '@patternfly/react-core';
 import { mount } from '@theforeman/test';
-import { AdvancedFields } from '../AdvancedFields';
-import { jobTemplateResponse } from '../../../__tests__/fixtures';
-
-jest.spyOn(patternfly, 'FormGroup');
-patternfly.FormGroup.mockImplementation(props => (
-  <div>{props.navAriaLabel}</div>
-));
-const mockStore = configureMockStore([]);
-const store = mockStore({
-  JOB_TEMPLATE: { response: jobTemplateResponse },
-});
+import { fireEvent, screen, render, act } from '@testing-library/react';
+import * as api from 'foremanReact/redux/API';
+import { JobWizard } from '../../../JobWizard';
+import * as selectors from '../../../JobWizardSelectors';
+import {
+  jobTemplateResponse as jobTemplate,
+  testSetup,
+  mockApi,
+} from '../../../__tests__/fixtures';
+
+const store = testSetup(selectors, api);
+mockApi(api);
+
+jest.spyOn(selectors, 'selectEffectiveUser');
+jest.spyOn(selectors, 'selectTemplateInputs');
+jest.spyOn(selectors, 'selectAdvancedTemplateInputs');
+
+selectors.selectEffectiveUser.mockImplementation(
+  () => jobTemplate.effective_user
+);
+selectors.selectTemplateInputs.mockImplementation(
+  () => jobTemplate.template_inputs
+);
+
+selectors.selectAdvancedTemplateInputs.mockImplementation(
+  () => jobTemplate.advanced_template_inputs
+);
```

Comment:
should also move to fixtures?

Reply:
I dont think they should as they are only relevant to the advanced step